pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1056,7 +1056,8 @@ cdef class _Timedelta(timedelta):
 
         See Also
         --------
-        Timestamp.isoformat
+        Timestamp.isoformat : Function is used to convert the given
+         Timestamp object into the ISO format.
```

Comment:
does this need to be indented?

Reply:
Yes, see here: https://pandas.pydata.org/docs/dev/development/contributing_docstring.html#section-5-see-also 
We typically indent it with 4 spaces as elsewhere.

Reply:
Will udpate this